mne/tests/test_epochs.py

```diff
@@ -1511,50 +1511,102 @@ def test_split_saving(tmp_path, split_size, n_epochs, n_files, size, metadata, c
             }
         )
         epochs.metadata = metadata
-    if concat:
-        epochs.drop_bad()
+    epochs.drop_bad()
+    if is_concat:
         epochs = concatenate_epochs([epochs[ii] for ii in range(len(epochs))])
-    epochs_data = epochs.get_data()
     assert len(epochs) == n_epochs
-    fname = tmp_path / "test-epo.fif"
-    epochs.save(fname, split_size=split_size, overwrite=True)
-    got_size = _get_split_size(split_size)
-    assert got_size == size
-    _assert_splits(fname, n_files, size)
-    assert not fname.with_name(f"{fname.stem}-{n_files + 1}{fname.suffix}").is_file()
-    for preload in (True, False):
-        epochs2 = mne.read_epochs(fname, preload=preload)
-        assert_allclose(epochs2.get_data(), epochs_data)
-        assert_array_equal(epochs.events, epochs2.events)
-
-    # Check that if BIDS is used and no split is needed it defaults to
-    # simple writing without _split- entity.
-    split_fname = tmp_path / "test_epo.fif"
-    split_fname_neuromag_part1 = tmp_path / f"test_epo-{n_files + 1}.fif"
-    split_fname_bids_part1 = tmp_path / f"test_split-{n_files + 1:02d}_epo.fif"
-
-    epochs.save(split_fname, split_naming="bids", verbose=True)
-    assert split_fname.is_file()
-    assert not split_fname_bids_part1.is_file()
-    for split_naming in ("neuromag", "bids"):
-        with pytest.raises(FileExistsError, match="Destination file"):
-            epochs.save(split_fname, split_naming=split_naming, verbose=True)
-    os.remove(split_fname)
-    # we don't test for reserved files as it's not implemented here
```

Comment:
I don't know what `reserved files` means here. I noticed, `Raw.save()` mentions something reserving the main file when producing splits. Is it something important I should be aware of? For now I just removed this comment.